website/ref/language.md

```diff
@@ -102,27 +104,79 @@ metacharacters; use quoted strings instead. For instance, to echo a star, write
 in line continuations; their use elsewhere is reserved will cause a syntax
 error.
 
-## Notes
+### Notes
 
 The three syntaxes above all evaluate to strings, and they are interchangeable.
 For instance, `xyz`, `'xyz'` and `"xyz"` are different syntaxes for the same
-string, and they are always equivalent.
+string, and they are always equivalent with the exception of
+**escape sequences** as documented above.
+
+## Number
+
+Elvish has a numeric data type, `float64`, created using this syntax:
+`(float64 $number)`. For convenience, all Elvish functions that
```

Comment:
> For convenience, all Elvish functions...

Please move the entirety of this discussion to builtin.md and link to it near the end of this section.